build-app-ng.lisp

```diff
@@ -0,0 +1,48 @@
+;;;; build-app-ng.lisp
+;;;;
+;;;; This file is loaded by the Makefile to produce a qvm-ng[.exe] binary.
+;;;;
+
+(unless *load-truename*
+  (error "This file is meant to be loaded."))
+
+(pushnew :hunchentoot-no-ssl *features*)
+(pushnew :drakma-no-ssl *features*)
```

Comment:
What's the deal with this no-ssl stuff?  Is this something we might want to reconsider for "qvm 2"?

Reply:
I've bumped into this before and thought it was an attempt to eliminate a dependency with OpenSSL.

Reply:
If that's the case, perhaps we could enable SSL for QMI / non-barebones SDK builds and selectively disable if dependencies aren't met.

Reply:
@jmbr is correct. This was mostly an issue on Windows, where my test VMs didn't have a suitable OpenSSL library available and I didn't want to have to worry about distributing it with our SDK. 

The feature that introduced this was https://github.com/rigetti/quilc/pull/144 and it is small enough that introducing another dependency feels wrong.

Reply:
That's reasonable. I will file an issue and label it qvm-ng as something to reconsider if we plan to have clients talking to "qvm-ng" over a network rather than localhost.

Reply:
That's a good point. It's something that is already do-able, but just not securely (which I'm sure is what has your spidey-sense tingling).

Reply:
This is now #152